reposerver/repository/repository.go

```diff
@@ -1102,9 +1102,24 @@ func runConfigManagementPlugin(appPath string, envVars *v1alpha1.Env, q *apiclie
 		defer func() { _ = closer.Close() }()
 		env = append(env, environ...)
 	}
-	env = append(env, q.ApplicationSource.Plugin.Env.Environ()...)
 	env = append(env, "KUBE_VERSION="+q.KubeVersion)
 	env = append(env, "KUBE_API_VERSIONS="+strings.Join(q.ApiVersions, ","))
+
+	parsedEnv := make(v1alpha1.Env, len(env))
+	for i, v := range env {
+		parsedVar, err := v1alpha1.NewEnvEntry(v)
+		if err != nil {
+			return nil, err
```

Comment:
What do you think we should return here? It would be nice to know the name of the env var that failed, but I suppose that's not really possible

I'm also wondering if it's even possible for this to happen

Reply:
I think the error might contain env variable defined in `ApplicationSource` only, not the local env variables of the repo-server pod. 

The `ApplicationSource` is not considered secret and already appears in repo-server logs, so I'm fine to keep it as is.